tests/test_registry.py

```diff
@@ -18,6 +23,8 @@ def test_get_all_tagged_releases():
 
 def test_get_releases_from_branch():
     """Test that all tagged releases of perticular branch (main) are returned."""
+    from aiidalab.registry.releases import _get_release_commits, _split_release_line
```

Comment:
The tests themselves will only run when you run pytest with `pytest -m registry`, but pytest will import this file always so you can't put the imports at the top, because they would fail.